packages/components/src/Menu/MenuItem.test.tsx

```diff
@@ -26,24 +26,32 @@
 
 import 'jest-styled-components'
 import React from 'react'
-import { assertSnapshot } from '@looker/components-test-utils'
+import { renderWithTheme } from '@looker/components-test-utils'
 
 import { MenuItem } from './MenuItem'
 
-test('MenuItem', () => {
-  assertSnapshot(<MenuItem>who!</MenuItem>)
-})
-test('MenuItem - icon', () => {
-  assertSnapshot(<MenuItem icon="Beaker">who!</MenuItem>)
+test('MenuItem renders', () => {
+  const { getByText } = renderWithTheme(<MenuItem>who!</MenuItem>)
+  expect(getByText('who!')).toBeVisible()
 })
+
 test('MenuItem - detail', () => {
-  assertSnapshot(<MenuItem detail="Is an excellent question">who!</MenuItem>)
+  const { getByText } = renderWithTheme(
+    <MenuItem detail="Is an excellent question">who!</MenuItem>
+  )
+  expect(getByText('Is an excellent question')).toBeVisible()
+})
+
+test('MenuItem - icon', () => {
+  const { getByText } = renderWithTheme(<MenuItem icon="Beaker">Icon</MenuItem>)
+  expect(getByText('Icon')).toBeVisible()
 })
 
-test('MenuItem - current', () => {
-  assertSnapshot(
-    <MenuItem current icon="Home">
-      who!
+test('MenuItem - artwork', () => {
+  const { getByText } = renderWithTheme(
+    <MenuItem iconArtwork={<svg xmlns="http://www.w3.org/2000/svg"></svg>}>
+      Artwork
     </MenuItem>
   )
+  expect(getByText('Artwork')).toBeVisible()
 })
```

Comment:
This test isn't proving that the SVG is rendering. Eek!

```suggestion
test('MenuItem - artwork', () => {
  renderWithTheme(
    <MenuItem iconArtwork={<svg xmlns="http://www.w3.org/2000/svg"><title>SVG Title Here</title></svg>}>
      Artwork
    </MenuItem>
  )
  expect(screen.getByTitle('SVG Title Here')).toBeVisible()
})
```